app/ui/src/main/java/com/fsck/k9/UiKoinModules.kt

```diff
@@ -26,6 +26,6 @@ val uiModules = listOf(
         fragmentModule,
         contactsModule,
         accountModule,
-        autodiscoveryModule,
+    autodiscoveryProvidersXmlModule,
```

Comment:
Please either match the indentation of the other entries or change all entries to use the new indentation.